reactive/INTRODUCTION_EN.md

```diff
@@ -0,0 +1,153 @@
+# Introduction to Reactive Programming
+
+> Written by [A~Z](https://github.com/AZ-0)
+
+Do you know of reactive programming? No? You should!
+
+Let's begin with wikipedia's obscure definition:
+
+## Definition
+
+The original article has the [french](https://fr.wikipedia.org/wiki/Programmation_r%C3%A9active) definition as well, but seeing as it is quite useless I just removed it.
+You shouldn't need french when programming anyway, no?
+
+We only have the [english article](https://en.wikipedia.org/wiki/Reactive_programming) left then, so here it is:
+> In computing, reactive programming is a declarative programming paradigm concerned with data streams and the propagation of change.
+> With this paradigm it is possible to express static (e.g., arrays) or dynamic (e.g., event emitters) data streams with ease, and also communicate that an inferred dependency
+> within the associated execution model exists, which facilitates the automatic propagation of the changed data flow.
+
+Wow, that's a lot of cool and complicated words! Don't worry, at this point not feeling overwhelmed would be concerning.
+(Or it just means you actually don't need this introduction at all so stop reading it and do something useful with your life)
+
+## An explanation to rekindle lost hopes
+
+Looks like you have a bad karma, because I am gonna be mean~. So mean that I'll use complicated words right away: `Publisher`! (Are you fearing me now?)
+
+> Haha, I do not fear *anything*! A Publisher is something that publish other things, no?
+
+**EXACT!** Or rather, a Publisher emits things. A Publisher emits values, but specially so.
+My goal here and what I'm not paid for is to make you understand just how this strange process works.
+I like exploding things, so we'll talk about a cannon.
+Don't worry about the `(x)` right now, you will understand them later on…
+
+Let's suppose you happily go to Wallmart to buy a cannon. It's a high-tech cannon that can automagically recharge.
+With your newfound super-duper cannon, you go back and `(1)` install it on a hill (locking on your boss), `(2)` gets the fuses batch, and `(3)` fire as many fuses as you want to throw cannonballs on your boss.
+- You don't know when the cannon will fire.
+- You don't even know whether the cannon will throw enough cannonballs to match your explosion desires (Has it enough ammunition? Will it explode halfway?)
+- You hate your boss enough that you decided to dance whenever the cannon throws a cannonball.
+- You somehow decided to run if the cannon explodes. Thanks, Captain Obvious.
+- You have decided to go on strike if your boss is still dead and you are running out of cannonballs. And if the cannon hasn't exploded, otherwise you would have fled too far~
```

Comment:
A typo supposedly + no-sense
```suggestion
- You have decided to go on strike if your boss is still alive and you are running out of cannonballs. And if the cannon hasn't exploded, otherwise you would have fled too far.
```